scenarios/two_party_countersigning/README.md

```diff
@@ -4,6 +4,20 @@
 
 This scenario tests the performance of `countersigning` operations.
 
+There are two roles, `initiate` and the `participate`. 
+
+The participants commit an entry to advertise that they are willing
+to participate in sessions. They listen for sessions and participate in one at a time. Three metrics are recorded:
```

Comment:
```suggestion
to participate in sessions. They listen for sessions and participate one at a time. Three metrics are recorded:
```

Reply:
I do mean "participate in" here, should this sentence be re-written to be clearer?

Reply:
Oh, re-reading it now I do get that. I guess it means "participate in one session at a time"? That makes sense now :slightly_smiling_face: